aptos-move/framework/aptos-framework/sources/reconfiguration.move

```diff
@@ -53,7 +53,7 @@ module aptos_framework::reconfiguration {
         system_addresses::assert_aptos_framework(aptos_framework);
 
         // assert it matches `new_epoch_event_key()`, otherwise the event can't be recognized
-        assert!(guid::get_next_creation_num(signer::address_of(aptos_framework)) == 1, error::invalid_state(EINVALID_GUID_FOR_EVENT));
+        assert!(guid::get_next_creation_num(signer::address_of(aptos_framework)) == 2, error::invalid_state(EINVALID_GUID_FOR_EVENT));
```

Comment:
why do we need to change to 2?  new_epoch_event_key is hard coded at https://github.com/aptos-labs/aptos-core/blob/5f3ac3920ce7fb9865f8357f04fe8c2fd93af769/types/src/on_chain_config/mod.rs#L158 and have many dependencies across codebase.

Reply:
changing all the other hard coded values too! I think it's bc we are adding a new event in account and it's created first, so we need to add 1 to all the other event keys